src/main/resources/generator/client/common/i18n/i18n.spec.ts

```diff
@@ -3,10 +3,12 @@ import { mergeTranslations } from '@/Translations';
 
 describe('i18n configuration', () => {
   it('loads en translation', () => {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
     expect(i18n.getResourceBundle('en', '')['home']['translationEnabled']).toBe('Internationalization enabled');
   });
 
   it('loads fr translation', () => {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
     expect(i18n.getResourceBundle('fr', '')['home']['translationEnabled']).toBe('Internationalisation activée');
```

Comment:
```suggestion
    expect((i18n.getResourceBundle('fr', '') as Translations).home.translationEnabled).toBe('Internationalisation activée');
```
with `import { mergeTranslations, type Translations } from '@/Translations';`